dwitter/static/main.css

```diff
@@ -207,8 +207,8 @@ ul.messages li.success{
   font-size: 13px;
   line-height: 25.28px;
   padding: .25em;
-  background: hsl(250, 56%, 95%);
-  color: hsl(250, 56%, 53%);
+  background: hsl(0, 0%, 92.5%);
```

Comment:
Did you mean to remove the purple tint of the background on the light theme?

Reply:
Yeah... it's still kind of present though, and it the end I only made it lighter...